content/docs/5.1/concepts/tikv-in-5-minutes.md

```diff
@@ -30,139 +32,143 @@ The following steps are only for test environments. For production environments
 
     ```bash
-    jshell --class-path tikv-client-java.jar:slf4j-api.jar --startup test_raw.java
-
-    Hello
-    [key: "k1"
-    value: "Hello"
-    , key: "k3"
-    value: "World"
-    ]
-    [key: "k1"
-    value: "Hello"
-    , key: "k2"
-    value: ","
-    , key: "k3"
-    value: "World"
-    , key: "k4"
-    value: "!"
-    , key: "k5"
-    value: "Raw KV"
-    ]
+    curl -o tikv-client-java.jar https://download.pingcap.org/tikv-client-java-3.2.0-SNAPSHOT.jar && \
+    curl -o slf4j-api.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.16/slf4j-api-1.7.16.jar
     ```
 
-### Python
-
-1. Install `tikv-client` python package
+2. Install `jshell`. The JDK version should be 9.0 or later.
+
+3. Try the `RAW KV` API.
+
+    1. Save the following script to the `test_raw.java` file.
+
+        ```java
+        import java.util.ArrayList;
+        import java.util.List;
+        import java.util.Optional;
+        import org.tikv.common.TiConfiguration;
+        import org.tikv.common.TiSession;
+        import org.tikv.kvproto.Kvrpcpb;
+        import org.tikv.raw.RawKVClient;
+        import org.tikv.shade.com.google.protobuf.ByteString;
+
+        TiConfiguration conf = TiConfiguration.createRawDefault("127.0.0.1:2379");
+        TiSession session = TiSession.create(conf);
+        RawKVClient client = session.createRawClient();
+
+        // put
+        client.put(ByteString.copyFromUtf8("k1"), ByteString.copyFromUtf8("Hello"));
+        client.put(ByteString.copyFromUtf8("k2"), ByteString.copyFromUtf8(","));
+        client.put(ByteString.copyFromUtf8("k3"), ByteString.copyFromUtf8("World"));
+        client.put(ByteString.copyFromUtf8("k4"), ByteString.copyFromUtf8("!"));
+        client.put(ByteString.copyFromUtf8("k5"), ByteString.copyFromUtf8("Raw KV"));
+
+        // get
+        Optional<ByteString> result = client.get(ByteString.copyFromUtf8("k1"));
+        System.out.println(result.get().toStringUtf8());
+
+        // batch get
+        List<Kvrpcpb.KvPair> list =client.batchGet(new ArrayList<ByteString>() {{
+        add(ByteString.copyFromUtf8("k1"));
+        add(ByteString.copyFromUtf8("k3"));
+        }});
+        System.out.println(list);
+
+        // scan
+        list = client.scan(ByteString.copyFromUtf8("k1"), ByteString.copyFromUtf8("k6"), 10);
+        System.out.println(list);
+
+        // close
+        client.close();
+        session.close();
+        ```
+
+    2. Run the script above to write "Hello World!" to TiKV:
+
+        ```bash
+        jshell --class-path tikv-client-java.jar:slf4j-api.jar --startup test_raw.java
+
+        Hello
+        [key: "k1"
+        value: "Hello"
+        , key: "k3"
+        value: "World"
+        ]
+        [key: "k1"
+        value: "Hello"
+        , key: "k2"
+        value: ","
+        , key: "k3"
+        value: "World"
+        , key: "k4"
+        value: "!"
+        , key: "k5"
+        value: "Raw KV"
+        ]
```

Comment:
Is this part the output or what we should write in the command?

Reply:
yes

Reply:
Is this the output?

OR

Is this what we should write in the command?

Reply:
it's output

Reply:
It would be better to separate the output and the command.

Reply:
agree